cba_settings_userconfig/cba_settings.sqf

```diff
@@ -227,19 +422,20 @@ force force ace_medical_statemachine_cardiacArrestTime = 120;
 force force ace_medical_statemachine_fatalInjuriesAI = 0;
 force force ace_medical_statemachine_fatalInjuriesPlayer = 1;
 force force ace_medical_treatment_advancedBandages = 2;
-force force ace_medical_treatment_advancedDiagnose = true;
+force force ace_medical_treatment_advancedDiagnose = 2;
```

Comment:
What are 0 1 and 2 in this case? Is 2 true?

Reply:
Advanced Diagnose: Checkbox to enable “Check Pulse”, “Check Blood Pressure” and “Check Response” treatment actions, if disabled CPR will only be shown when performing CPR is necessary. This setting is imperative to determining if a patient is unconscious or in cardiac arrest.

How is this a checkbox if it has 0, 1 and 2 as a value? I have to go in game and actually check them out.

Reply:
<img width="402" alt="image" src="https://user-images.githubusercontent.com/57898126/147141610-65861375-cdaa-4e3d-9971-3a3905053361.png">
These are the options that reflect 0,1 and 2. 2 sounds the best so I agree :)